<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowLEDControl16GForm</class>
 <widget class="QWidget" name="ShowLEDControl16GForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light</string>
  </property>
  <widget class="QToolButton" name="toolButtonAllON">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>All ON</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>DipSW</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>231</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>24</number>
   </attribute>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Target</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Now</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonSave">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetDipSW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPatternNo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
